source/geh_common/src/geh_common/data_products/electricity_market_measurements_input/capacity_settlement_metering_point_periods_v1.py

```diff
@@ -8,14 +8,14 @@
 
 schema = T.StructType(
     [
-        # ID of the consumption metering point (parent)
+        # ID of the consumption metering point (parent) OR D22 child metering point
         T.StructField("metering_point_id", T.StringType(), not nullable),
         #
-        # Date when the consumption metering is either (a) entering 'connected'/'disconnected' first time or (b) move-in has occurred.
+        # Date when the consumption metering or D22 child metering point is either (a) entering 'connected'/'disconnected' first time or (b) move-in has occurred. on E17 metering point
```

Comment:
I thought this part would be unchanged, and that it was the `child_metering_point_id` that needed to be updated?